shell/packages/sandstorm-accounts-ui/login_buttons.js

```diff
@@ -2,6 +2,9 @@
 var loginButtonsSession = Accounts._loginButtonsSession;
 
 var helpers = {
+  currentRoute: function (routeName) {
```

Comment:
(I recognize I goofed this up in the sandstorm-ui-topbar package, and you were probably just following that bad example.  I should probably fix it there too.)